paasta_tools/paasta_cli/cmds/rollback.py

```diff
@@ -53,22 +54,47 @@ def add_subparser(subparsers):
     list_parser.set_defaults(command=paasta_rollback)
 
 
+def validate_given_instances(service, args_instances):
+    """Trying to find which instances are actually valid for the given service"""
+    service_instances = list_all_instances_for_service(service)
+    invalid_instances = None
+
+    if not args_instances:
```

Comment:
when is this condition true? isn't args_instances going either be 'main' or some other user input?